ibm/data_source_ibm_cis_healthchecks.go

```diff
@@ -115,6 +115,26 @@ func dataSourceIBMCISHealthChecks() *schema.Resource {
 							Type:     schema.TypeInt,
 							Computed: true,
 						},
+						cisGLBHealthCheckHeader: {
+							Type:     schema.TypeSet,
```

Comment:
It is advisable to have typeList for datasources..
Please chage it to typeSet in datasource as it is easily accessable..

Reply:
it did not work after changing it to TypeList

Reply:
Hi..
When changed to type list `Set: hashByMapKey(cisGLBHealthCheckHeadersHeader),` has to be removed but schema
seperate flatten function should be written for datasource that returns list

Reply:
modified